### ISSUE TITLE
TICA *2

### ISSUE THREAD

**Issue by franknoe** [markovmodel/PyEMMA#385]
Why are all the quantities in TICA multiplied by 2? E.g.:

``` python
    self._N_cov_tau += 2.0 * np.shape(Y)[0]
```

I guess this has to do with the different handling for `_force_eigenvalues_le_one=True`, but makes the code hard to understand. Even if is it correct because you multiply everything by 2, is it still correct to do 

``` python
    self.cov /= self._N_cov - 1
```

in param_finish. Shouldn't that then be -2 ?

**Comment by fabian-paul** [comment]
That's true. It should be self._N_cov - 2, indeed. Thanks for finding the mistake.
The rest is a matter of taste, we can either multiply all counts by 2 or we will need counts of 0.5 for the "tails" of the time series (with force_eigenvalues_le_one).
I'm fine with any of the two choices.

**Comment by fabian-paul** [comment]
Fix proposed in PR #386.
